src/layer/tile/GridLayer.js

```diff
@@ -229,6 +229,8 @@ export var GridLayer = Layer.extend({
 	redraw: function () {
 		if (this._map) {
 			this._removeAllTiles();
+			this._tileZoom = this._clampZoom(this._map.getZoom());
+			this._updateLevels();
```

Comment:
```suggestion
			var tileZoom = this._clampZoom(this._map.getZoom());
			if (tileZoom !== this._tileZoom) {
				this._tileZoom = tileZoom;
				this._updateLevels();
			}
```

Reply:
Hi @johnd0e
I don't remember the context of this PR, but feel free to update it if there is space for improvements!